### DISCUSSION TITLE
Include shared in album in timeline/map/face recognition

### DISCUSSION THREAD

**Discussion by Deedikjupijn** [immich-app/immich#13421] [Feature Request]
**The feature**

I would like the ability to share images and albums to other Immich users with all the metadata included, this would then make sure that they are able to see these images as if they uploaded it (and have their accounts run facial recognition against their saved faces). 

The reason for this would be that I have a bunch family pictures that I would love to share to my family while not sharing any other pictures from my account to them. You can already share albums or connect users but either are not the right solution just yet, the first allows them to see the pictures and iswhat I'm using now. The issue is that when those users want to search for an image they are unable to because the images from the shared albums to not appear in their timeline map or anything else. The latter would share every image and that's not what I wish to do either. 

If anyone has an idea for how to achieve this, I'm all here for it!

Thanks for taking ur time to read this and as always,

Hope you have an amazing day!

**Platform**

- [x]  Server

- [x]  Web

- [x]  Mobile

(Edit made it more readable)

**Comment by bo0tzz** [comment]
Tracked in #12614

**Comment by txemi** [comment]
This request is very close to what I’m looking for, but it bundles several behaviors (timeline, map, face recognition, metadata sharing).

I’ve created a narrower proposal that focuses only on one aspect: letting the receiving user run their own face recognition pipeline on shared albums/assets, while NOT sharing ML/People data between accounts:

https://github.com/immich-app/immich/discussions/24290

Maybe this smaller, well‑scoped feature could be an earlier step towards the broader behavior described here.
